node-manager/src/ldkstorage.rs

```diff
@@ -238,6 +240,14 @@ impl MutinyNodePersister {
             self.storage.get(key).map_err(MutinyError::read_err);
         deserialized_value.ok()
     }
+
+    pub(crate) fn list_payment_info(&self, inbound: bool) -> Vec<(String, PaymentInfo)> {
+        let prefix = match inbound {
+            true => PAYMENT_INBOUND_PREFIX_KEY,
+            false => PAYMENT_OUTBOUND_PREFIX_KEY,
+        };
+        self.storage.scan(prefix, None)
```

Comment:
matching on a boolean is a little weird but nbd

Reply:
keeping it for now